### PR TITLE
TYPO3 11 LTS compatibility

### LINKED ISSUE #36: TYPO3 11 LTS compatibility
It should be possible to install this extension on v11 LTS.

### PR DESCRIPTION
**Pull request by h3nn3s**
resolves #36

### CODE REVIEW

File: Configuration/TCA/Overrides/sys_template.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-defined('TYPO3_MODE') || die();
+defined('TYPO3') || die();
```
Comment (xperseguers) [inline Configuration/TCA/Overrides/sys_template.php:3]: I doubt this will work in older versions of TYPO3.

Comment (h3nn3s) [reply Configuration/TCA/Overrides/sys_template.php:3]: damn, you are right...

**Review by xperseguers — APPROVED** [review]
This looks OK, I'll merge it and test it for a while in the wild ;-)

**Comment by h3nn3s** [comment]
Thank you for quick merging.

**Pull request merged**